### DISCUSSION TITLE
2024-05-30(목)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#68] [데일리스크럼]
아래 내용을 복사해서 댓글에 작성해주시면 됩니다.

```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by minyoung0503** [comment]
### 완료한 일
- 리뷰박스 컴포넌트 구현

### 해야할 일
- 상품 상세 페이지 -> 리뷰 UI 작업

### 어려운 일

**Comment by Seoin02** [comment]
### 완료한 일
- 마이(로그인x) UI 작업
- 약관 내용 정리
- 뒤로가기 버튼 구현

### 해야할 일
- 회원가입 UI 구현
- 온보딩 UI 구현

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- PC 화면에서 모달이 레이아웃을 이탈하는 현상 해결
- AWS CI/CD 버그 조치

### 해야할 일
- 카테고리 버튼 컴포넌트 구현
- 스크롤 Top 버튼 컴포넌트 구현
- 배너 캐러셀 컴포넌트 구현

### 어려운 일
- 파이팅!

**Comment by q45402sk** [comment]
### 완료한 일
card 컴포넌트 코드리뷰 반영

### 해야할 일
zzim 컴포넌트

### 어려운 일

**Comment by seolsis** [comment]
### 완료한 일
- 장바구니 기능 구현, UI 마무리 
### 해야할 일
- 결제 페이지 토스뱅크, 카카오페이 연동
### 어려운 일
- 페이지간 데이터 전달 => 전역상태관리
